code/datums/components/mood.dm

```diff
@@ -82,7 +82,7 @@
 			var/datum/mood_event/event = mood_events[i]
 			msg += "[event.description]\n"
```

Comment:
Would it be possible to filter out the 0 change moodlets at this level and then add a seperate section below moodlets called "thoughts" or something like that?

Reply:
managed to do it, but it was a bit hacky. Not sure if it is up to coding standards
Only other way to do it would be making 0 mood moodlets something completely seperate from moodlets